### ISSUE TITLE
Support language

### ISSUE THREAD

**Issue by luckydonald** [ZeevG/python-forecast.io#25]
Under the Docs for v2 is an `lang=`  Option.
It would be nice to see this implemented as something like this:

``` python
foo = forecastio.load_forecast(key, latitude, longitude, lang="en")
```

This was added to the API on 29 May 2014:
 https://developer.forecast.io/docs/v2#options

Currently this is not working. Are there any quick workarounds?

**Comment by suhajdab** [comment]
+1

**Comment by dillenmeister** [comment]
+1

**Comment by MicheleCatalano** [comment]
+1

**Comment by fogi** [comment]
+1

**Comment by arhangel66** [comment]
+1

**Comment by DarkFox** [comment]
+1

**Comment by ZeevG** [comment]
Thanks for everyone's patience and apologies it took so long to add this feature. I've now released version 1.4.0 which allows you to specify a language.
